Pandas.md

```diff
@@ -0,0 +1,327 @@
+# Import
```

Comment:
CheatSheet metadata is missing, please refer other cheat sheets to see how to add that. Also please use all lowercase letters for file name

Reply:
okk